docs/ABI/TypeMetadata.rst

```diff
@@ -85,14 +88,18 @@ contain the following fields:
 
 - The `nominal type descriptor`_ is referenced at **offset 1**.
 
-- A reference to the **parent** metadata record is stored at **offset 2**. For
-  enums that are members of an enclosing nominal type, this is a reference to
-  the enclosing type's metadata. For top-level enums, this is null.
+- If the enum is generic, then the
+  `generic argument vector`_ begins at **offset 2**.
+
+- In certain circumstances, there is a `payload size`_ immediately following
```

Comment:
I think 'certain circumstances' are the enum has more than one payload case, and at least one payload case has a size that depends on a generic parameter or is resilient.

Reply:
Do you think that's an important enough detail to include here?  I thought it was obscure enough to just leave it vague.  Honestly, I think it's arguably a private implementation detail more than something that needs documentation.